pkgs/by-name/wi/windmill/package.nix

```
@@ -158,6 +158,10 @@ rustPlatform.buildRustPackage {
     description = "Open-source developer platform to turn scripts into workflows and UIs";
     homepage = "https://windmill.dev";
     license = lib.licenses.agpl3Only;
+    sourceProvenance = with lib.sourceTypes; [
+      binaryNativeCode
```

Given your line of reasoning:

```suggestion
      binaryNativeCode # librusty_v8
```

---

I'd say it makes more sense on librusty itself, for clarity as well as keeping the meta information associated with the most relevant derivation

---

It would make sense if librusty is an element of input set.